src/Roslyn.Suppressions/Roslyn.Suppressions/SuppressionExecutionContext.cs

```diff
@@ -55,5 +55,7 @@ private void GetProjectKey(AnalyzerOptions analyzerOptions)
         public bool IsInConnectedMode => SettingsKey != null;
 
         public string SettingsKey { get; private set; } = null;
+
+        public string Mode => SettingsKey != null ? "Connected" : "Standalone";
```

Comment:
Can we maybe do this conversion in the ETW method and pass along the boolean IsInConnectedMode? It feels odd to add this extra field to the class when it's used only for logging.

Reply:
I thought of that, but I think this would make more sense from readability point and also we would not have context logic in ETW method.